### PR TITLE
Add participant view

### PR DESCRIPTION
**Pull request by marcauberer**
- Add participant view

### CODE REVIEW

File: src/theme.dark.less [hunk 1]
```diff
@@ -37,6 +37,10 @@
   }
 }
 
+.header-logo {
+  background: #0000;
```
Comment (Ordinateur-Hack) [inline src/theme.dark.less:41]: you've used `@white` in another place, can you use `@black` here?

File: src/theme.light.less [hunk 2]
```diff
@@ -36,6 +36,31 @@
...
+  }
+}
+
+.poll-name {
+  position: absolute;
+  left: 50%;
+  transform: translate(-50%, -85%);
+  font-weight: bolder;
+  color: white;
+  font-family: "Agency FB", sans-serif;
```
Comment (Ordinateur-Hack) [inline src/theme.light.less:61]: pls change that font later on 😅

**Review by Ordinateur-Hack — APPROVED** [review]

File: src/app/pages/poll-participants/poll-participants.component.ts [hunk 3]
```diff
@@ -2,17 +2,51 @@
...
+    position: 1,
+    type: ItemType.MultipleChoice,
+    answers: [
+      { selectionOption: 'Option 1', answerCount: 1 },
+      { selectionOption: 'Option 2', answerCount: 10 },
+      { selectionOption: 'Option 3', answerCount: 4 }
+    ]
+  };
+  activeItemType: 'multiple-choice';
+  answer = '';
```
[the remark this answers is not in the document]
Comment (marcauberer) [reply src/app/pages/poll-participants/poll-participants.component.ts:33]: Yes, to mock a multiple choice item until the web socket communication is ready to work with.

File: src/theme.dark.less (hunk 1 above)
Comment (marcauberer) [reply src/theme.dark.less:41]: This is transparent. But yes, I can use a variable like everywhere else

File: src/theme.light.less (hunk 2 above)
[the remark this answers is not in the document]
Comment (marcauberer) [reply src/theme.light.less:61]: @Ordinateur-Hack which font would you recommend? You're way better than me in terms of design ;)

File: src/app/pages/poll-participants/poll-participants.component.ts [hunk 4]
```diff
@@ -2,17 +2,51 @@
...
+   *
+   * @param route Active route
+   */
+  constructor(
+    private route: ActivatedRoute
+  ) {
+    this.route.params.subscribe( params => console.log(params));
 
+    // Connect to WebSocket
+    this.initializeWebSocketConnection();
```
[the remark this answers is not in the document]
Comment (marcauberer) [reply src/app/pages/poll-participants/poll-participants.component.ts:51]: @Ordinateur-Hack  Changed it. Is it best practise to not use the constructor?

**Pull request merged**